cmd/errtrace/main.go

```diff
@@ -880,11 +883,14 @@ func optoutLines(
 ) map[int]int {
 	lines := make(map[int]int)
 	for _, cg := range comments {
-		for _, c := range cg.List {
-			if strings.Contains(c.Text, "//errtrace:skip") {
-				lineNo := fset.Position(c.Pos()).Line
-				lines[lineNo] = 0
-			}
+		if len(cg.List) > 1 {
+			continue // skip multiline comments
```

Comment:
suggest adding context on why we skip multiline comments. added what i think is the reason.

```suggestion
			// skip multiline comments which are full line comments, not tied to a return.
			continue
```